examples/Thermostat/Device/Program.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc;
+using IceRpc.Slice;
+using Igloo;
+using Microsoft.Extensions.Logging;
+using ThermostatDevice;
+
+using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+    builder
+        .AddSimpleConsole()
+        .AddFilter("IceRpc", LogLevel.Debug));
+
+var thermoBot = new ThermoBot();
```

Comment:
Maybe `IglooThermostat` instead of `ThermoBot`. I find the ThermoBot name a bit confusing.

Reply:
It should be ThermoSomething. 

I first wanted to use ThermoDevice, but the namespace is already ThermostatDevice so it's too repetitive.

I picked ThermoBot because it's the actual service (just like ChatBot in the Greeter examples) and it's a fake / (ro)bot. Can you think of a better name?